container/root/usr/local/bin/archstring

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Script to detect processor architecture and output one of the two input parameters
+# Example: when a download path requires an arch-specific package name
+#
+# Param 1: output if an x86_64 architecture is detected
+# Param 2: output if an arm64 or aarch64 architecture is detected
+
+REQUIRED_PARAM_COUNT=2
+
+if [[ $# != $REQUIRED_PARAM_COUNT ]]; then
+  echo "error: ${REQUIRED_PARAM_COUNT} parameters are required"
+  exit 1
+fi;
+
+DETECTED_ARCH="$(uname -m)"
+ARCH_X64='x86_64'
+ARCH_ARM64='aarch64'
+ARCH_ARM64_ALT='arm4';
```

Comment:
typo: should be **arm64** not arm4

Reply:
perils of late-night coding :) 
testing the negative case!